src/components/Project/project.scss

```diff
@@ -50,6 +50,79 @@
         margin-left: 10px;
       }
 
+      &__source {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        min-height: 26px;
+
+        .input-wrapper {
+          width: 100%;
+        }
+
+        .input {
+          width: 100%;
+        }
+
+        &-edit {
+          position: relative;
+          display: none;
+          margin-left: 10px;
+          text-decoration: underline;
+          cursor: pointer;
+        }
+
+        &-text {
+          max-width: 230px;
+          color: $cornflowerBlue;
+        }
+
+        &:hover {
+          .general-info__source-edit {
+            display: block;
+          }
+        }
+      }
+
+      &__labels {
+        .chips-wrapper {
+          flex-wrap: wrap;
+        }
+
+        &-item {
+          display: inline-flex;
+          margin: 10px 0 0 10px;
+          padding: 5px 10px;
+          color: $primary;
+          font-size: 14px;
+          border: $labelBorder;
+          border-radius: 15px;
+        }
+
+        .job-labels__item-icon-close {
+          transform: scale(0.7);
+        }
+
+        .job-labels__button-add {
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          width: 30px;
+          height: 30px;
+          margin: 10px 0 0 10px;
+          border: 1px solid #a44cc5;
```

Comment:
look for border in border.scss or add them there. and look for colors in colors.scss, add colors to this file if you need